python/paddle/fluid/layers/nn.py

```diff
@@ -9334,6 +9335,76 @@ def get_tensor_from_selected_rows(x, name=None):
     return out
 
 
+def shuffle_channel(x, group, name=None):
+    """
+    **Shuffle Channel Operator**
+    This operator shuffles the channels of input x.
+    It divide the input channels in each group into :attr:`group` subgroups,
+    and obtain a new order by selecting element from every subgroup one by one.
```

Comment:
what do you mean by "obtain a new order by selecting element from every subgroup one by one"? Is the new order random?

Reply:
No, it's not random.
Step1: divide the origin channels to G subgroups equally.
Step2: Select i-th element from every subgroup, (i from 1 to C/G)
Step3: Reorder
According to the paper: "first, divide the channels in each group into several subgroups, then feed each group in the next layer with different subgroups"